frontend/app/src/components/helper/display/icons/AssetIcon.vue

```diff
@@ -117,6 +100,20 @@ const currency = computed<string | undefined>(() => {
     ?.unicodeSymbol;
 });
 
+const { assetSymbol, assetName, assetInfo } = useAssetInfoRetrieval();
+
+const symbol = computed(() =>
+  get(assetSymbol(get(identifier), get(enableAssociation)))
+);
+const name = computed(() =>
+  get(assetName(get(identifier), get(enableAssociation)))
+);
+const asset = computed(() =>
+  get(assetInfo(get(identifier), get(enableAssociation)))
```

Comment:
no need to wrap assetInfo to computed you can do `const asset = assetInfo(identifier, enableAssociation)`;

See how identifier, and enableAssociation have not `get` this is because they are now declared as `MaybeRef` and the `get` happens inside asset info (enable association had a bug it will be fixed in my PR). It should make the code a bit cleaner. 

Also note whenever we can let's do that to computed methods in stores.